tests/zv/build.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2019-2025 The OpenEBS Authors
```

Comment:
2019? This is a new file
Also now we don't need to and shouldn't add headers IIRC

Reply:
removed